tests/unit/ComponentHelper.test.js

```diff
@@ -71,17 +76,18 @@ suite('ComponentHelper', function () {
     });
 
     test('get position of root and child', function(done) {
-      var el = addEntityWithComponents({
+      var el = createEntityWithComponents({
         'position': { x: 1, y: 2, z: 3 },
       });
-      var child = addEntityWithComponents({
+      var child = createEntityWithComponents({
         'position': { x: 5, y: 5, z: 1 },
       });
       child.className = 'child';
       el.appendChild(child);
+      scene.appendChild(el);
       var componentsToCheck = ['position', { selector: '.child', component: 'position' }];
 
-      utils.whenEntityLoaded(child, function() {
+      utils.whenEntityLoaded(el, function() {
```

Comment:
Running the tests in both Firefox and Chrome (via karma) uncovered a race condition with this suite. Chrome would report that the child entity was not loaded even though the parent entity reports as loaded. Vice versa in Firefox. Something to do with the specifics of the web component implementation/polyfill, I suspect. 

Both browsers behave identically if you force the order of operations by constructing the DOM tree fully, before finally adding it to the scene.